packaging/centreon-engine-daemon-debuginfo.yaml

```diff
@@ -0,0 +1,67 @@
+name: "centreon-engine-daemon-debuginfo"
+arch: "${ARCH}"
+platform: "linux"
+version_schema: "none"
+version: "${VERSION}"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  Debuginfo package for centreon-engine-daemon.
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://centreon.com"
+license: "Apache-2.0"
+
+contents:
+  - src: "../bin/centengine.debug"
+    dst: "/usr/lib/debug/usr/sbin/centengine.debug"
+    file_info:
+      mode: 0644
+
+  - src: "../bin/centenginestats.debug"
+    dst: "/usr/lib/debug/usr/sbin/centenginestats.debug"
+    file_info:
+      mode: 0644
+
+  - src: "../lib/externalcmd.so.debug"
+    dst: "/usr/lib/debug/usr/lib64/centreon-engine/externalcmd.so.debug"
+    file_info:
+      mode: 0644
+
+overrides:
+  rpm:
+    depends:
+      - centreon-engine-daemon = ${VERSION}-${RELEASE}${DIST}
+    conflicts:
+      - centreon-engine-debuginfo
+      - centreon-engine-extcommands-debuginfo
```

Comment:
conflicts, replace and provides. All three on the same dependencies ?

Reply:
yes it's needed to properly remove old package and install new one